### ISSUE TITLE
Y muscle menu broke for me  [Bug]: 

### ISSUE THREAD

**Issue by xMrGodz** [CMGStudios/issue-tracker#450]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

https://youtu.be/9Bb1CjTwSj8 Last couple min in video will show you

### Expected behaviour

to be able to remove clothing items

### Steps to reproduce

I dont know i rest game and keybinds many times nothing seems to work been like this for 1 or 2 weeks

### CMG User ID

286579

**Comment by crazychris2404** [comment]
Any update @xMrGodz

**Comment by xMrGodz** [comment]
Yes it worked when I changed my mouse input however when changing my mouse is not as smooth as I like it. Is there any recommendations on what to do?

**Comment by crazychris2404** [comment]
> Yes it worked when I changed my mouse input however when changing my mouse is not as smooth as I like it. Is there any recommendations on what to do?

What exactly do you mean by not smooth can you attach a video please
